### ISSUE TITLE
[model-rewrite] IBertModel: data-dep branch on quant activation range (27 breaks)

### ISSUE THREAD

**Issue by penguinwu** [penguinwu/oss-model-graph-break-corpus#86]
## Summary

IBertModel produces 27 graph breaks (across eval + train) from a single quant-range check pattern.

Sub-pattern of #54 (umbrella). Filed separately for a focused target.

## Pattern

`transformers/models/ibert/quant_modules.py:173` and `:178`:

```python
if self.x_min.min() > -1.1e-5 and self.x_max.max() < 1.1e-5:
    self.x_min = self.x_min - 1
    self.x_max = self.x_max + 1
```

The branch reads two running stats (`x_min`, `x_max`) and compares against a hard-coded epsilon. Dynamo can't trace through the data-dep comparison.

## Fix direction (model rewrite)

This is a quant-aware-training warmup hack — the branch handles the case where stats haven't been initialized yet. Two HF-side options:

1. Make the update branchless using `torch.where`: always compute `x_min - 1` and `x_max + 1`, select via `torch.where(condition, new_val, old_val)`.
2. Skip the warmup entirely in eval mode (or after stats reach a stable epsilon-distance from zero), since this only matters during early training.

## Affected Models

| Model | Modes | Breaks |
|-------|-------|--------|
| IBertModel | train | 27 |

## Impact

Eliminates 27 breaks. iBert has additional breaks beyond this — won't flip to fullgraph alone.

## Related

- #54 (umbrella, being broken up)
- #77, #78, sister Encodec issue (same umbrella)

## Tested On

- PyTorch 2.10.0a0+gitf32431e (PR #181552)
- Transformers 5.6.2
- Sweep: 2026-04-28 `animesh-fullgraph`
